_includes/callback-reference.md

```diff
@@ -76,20 +81,14 @@ three example scenarios of why this is important:
         "number": 222222222
     },
     "transaction": {
-        "id": "/psp/{{ api_resource }}/payments/{{ page.payment_id }}/<transaction type>/{{ page.transaction_id }}",
+        "id": "/psp/{{ api_resource }}/payments/{{ page.payment_id }}/authorizations/{{ page.transaction_id }}",
         "number": 333333333
     }
 }
 ```
 
 {% endif %}
 
-{:.table .table-striped}
```

Comment:
Let's add examples of a response for an authorization request here. I think we have an include for that.

Reply:
added!